core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -18,6 +18,18 @@
   admin-navbar .oppia-navbar-admin {
     top: 0;
   }
+  admin-navbar .oppia-navbar-container {
+    background-color: #00376d !important;
+  }
+  admin-navbar .oppia-dropdown-item-text {
+    color: #00376d !important;
```

Comment:
Avoid using `important`, as suggested [in the wiki page](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

Reply:
Sean has suggested a way to avoid doing this via the id: [comment link](https://github.com/oppia/oppia/pull/10430#discussion_r478421569)

You can follow that comment and remove the `id` attribute. [I'll approve this PR once it's done as other codeowner files LGTM!]